layouts/partials/navbar.html

```diff
@@ -11,7 +11,7 @@
       <a class="navbar-item" href="{{ $home }}">
         {{ if $logo }}
         {{ $url := printf "img/logos/%s" $logo | relURL }}
-          <img src="{{ $url }}" width="200">
+          <img src="{{ $url }}" width="85">
```

Comment:
😬 

Sorry nate, this was me, moving too fast and not cleaning up after myself.

Would it be possible to add a `class="blah"` to this instead and then set the width via the `_custom.sass` file instead?